README.md

```diff
@@ -1 +1,9 @@
 # cs-5666-scatt-gp
+
+## Demo Instructions
+* Login to the Appalachian student server.
```

Comment:
It's not a big deal but you can actually clone this anywhere with git installed, it's not specific to the ASU student server.